src/core/models/outputs/output_basic.h

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 IOTA Stiftung
```

Comment:
```suggestion
// Copyright 2022 IOTA Stiftung
```

Reply:
renaming files will not bump the copyright year, no?